analyzers/src/SonarAnalyzer.Common/Helpers/SyntaxNodeExtensions.cs

```diff
@@ -37,6 +37,22 @@ public static bool ToStringContainsEitherOr(this SyntaxNode node, string a, stri
         return toString.Contains(a) || toString.Contains(b);
     }
 
+    public static string GetMappedFilePath(this SyntaxNode root)
```

Comment:
I'm a bit concerned about this method from SyntaxNodeExtension. 
Since it is named as such, people might mistakenly rely on it and call it with non-root syntaxNodes, expecting to get the original path. I cannot find a simple way to restrict the input parameters to only roots, but what do you think about renaming it to "GetMappedFilePathFromRoot"?
This might help clarify its purpose